mapping-for-navigation/_posts/0000-01-08-implicit-maneuvers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Mapping implicit turn-restriction
+---
+
+
+## What are implicit turns?
+
+Turns on highways should never be sharp **less than ~90 degree**. An implicit turn is a turn which is legal with respect to junction modeling but is forbidden by the algorithm to avoid unwanted and dangerous turn recommendations. These junction have high chances of missing turn-restriction and adding these restriction makes routing engine to avoid such turns when calculating the route.
```

Comment:
Not sure we should hard-code the 90-degrees here - it depends on factors such as highway class, maxspeeds, road width, etc.